.github/workflows/attach_release_assets.yml

```diff
@@ -0,0 +1,37 @@
+# Workflow that attaches additional assets to github release.
+name: Attach Release Assets
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  trigger-build:
+    uses: ./.github/workflows/build.yml
+
+  trigger-dfu-check:
+    uses: ./.github/workflows/dfu_check.yml
+
+  trigger-target-test:
+    uses: ./.github/workflows/on_target.yml
+
```

Comment:
Why do we test this again?

Reply:
Currently the on_target workflow is only once every night (not for every commit). Hence I invoke it to ensure that the latest commit is tested while releasing.